MSAL/test/unit/MSALResultTests.m

```diff
@@ -86,8 +86,8 @@ - (void)testMSALResultWithTokenResult_whenTokenResultContainsNilAuthority_should
     MSALResult *result = [MSALResult resultWithTokenResult:tokenResult error:&error];
     
     XCTAssertNil(result);
-    XCTAssertEqualObjects(error.domain, @"MSALErrorDomain");
-    XCTAssertEqual(error.code, -42000);
+    XCTAssertEqualObjects(error.domain, @"MSIDErrorDomain");
+    XCTAssertEqual(error.code, MSIDErrorInvalidDeveloperParameter);
```

Comment:
Why would we have MSID error in MSALResult?

Reply:
We decided to always create MSID errors in MSAL code. Mapping from MSID error to MSAL error should be only in public classes before we return the error to the developer.